tests/Unit/Campaigns/DataTransferObjects/CampaignGoalDataTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Give\Tests\Unit\Campaigns\DataTransferObjects;
+
+use Give\Campaigns\DataTransferObjects\CampaignGoalData;
+use Give\Campaigns\Models\Campaign;
+use Give\DonationForms\Models\DonationForm;
+use Give\Donations\Models\Donation;
+use Give\Donors\Models\Donor;
+use Give\Framework\Database\DB;
+use Give\Tests\TestCase;
+use Give\Tests\TestTraits\RefreshDatabase;
+use PHPUnit_Framework_MockObject_MockBuilder;
+
+/**
+ * @unreleased
+ */
+final class CampaignGoalDataTest extends TestCase
```

Comment:
Is there any specific reason to use the final statement here?

Reply:
The `final` statements came with the copy/paste. It isn't necessary, but the purpose of `final` is to prevent inheritance, which _is_ appropriate for a test case.